### ISSUE TITLE
How can we retrain the model on custom dataset (Indian Faces)? 

### ISSUE THREAD

**Issue by guptakvgaurav** [nyoki-mtl/keras-facenet#18]
I would like to re-train the model on custom data-set. How can we achieve that?

**Comment by mchugh16** [comment]
Did you figure this out?

**Comment by Abhiyendra94** [comment]
Kind of

**Comment by guptakvgaurav** [comment]
@mchugh16 Yes, I retrained the original model from [davidsandberg's](https://github.com/davidsandberg/facenet) repo ([find re-trainining steps here](https://github.com/davidsandberg/facenet/wiki/Train-a-classifier-on-own-images#train-a-classifier-on-your-own-dataset)), and converted it into kera model using `tf_to_keras.ipynb` utility code provided in this repo. This approach worked for me.

**Comment by hardikprakash** [comment]
> 

Would you be willing to share this trained model, if you still have it?

**Comment by lynnwilliam** [comment]
> I would like to re-train the model on custom data-set. How can we achieve that?

Can you share the source indian dataset of faces ?
